ray-operator/controllers/ray/rayservice_controller.go

```diff
@@ -672,8 +677,6 @@ func decideClusterAction(ctx context.Context, rayServiceInstance *rayv1.RayServi
 func (r *RayServiceReconciler) updateRayClusterInstance(ctx context.Context, rayClusterInstance *rayv1.RayCluster) error {
 	logger := ctrl.LoggerFrom(ctx)
 	logger.Info("updateRayClusterInstance", "Name", rayClusterInstance.Name, "Namespace", rayClusterInstance.Namespace)
-	// Printing the whole RayCluster is too noisy. Only print the spec.
-	logger.Info("updateRayClusterInstance", "rayClusterInstance.Spec", rayClusterInstance.Spec)
```

Comment:
`rayClusterInstance.Spec` is very verbose and I find it is not useful for debugging.